lighthouse-core/config/config.js

```diff
@@ -656,6 +656,19 @@ class Config {
       }
     });
 
+    // The `full-page-screenshot` audit belongs to no category, but we still want to include
+    // it for the categories that utilize it.
+    if (settings.onlyCategories) {
```

Comment:
What if `onlyAudits` is set? In that situation, `full-page-screenshot` should be explicitly listed in `onlyAudits` right?

Reply:
yeah that's my understanding